backend/src/infra/security/token.manager.ts

```
@@ -49,11 +71,53 @@ export class TokenManager {
+      return;
+    }
+
+    const secretService = SecretService.getInstance();
+    const [existingPrivateKey, existingPublicKey, existingKid] = await Promise.all([
+      secretService.getSecretByKey(JWT_SIGNING_PRIVATE_KEY_SECRET),
+      secretService.getSecretByKey(JWT_SIGNING_PUBLIC_KEY_SECRET),
+      secretService.getSecretByKey(JWT_SIGNING_KID_SECRET),
+    ]);
+
+    if (existingPrivateKey && existingPublicKey) {
+      this.setSigningKeys(existingPrivateKey, existingPublicKey, existingKid || crypto.randomUUID());
```

ESLint

[error] Replace `existingPrivateKey,·existingPublicKey,·existingKid·||·crypto.randomUUID()` with `⏎········existingPrivateKey,⏎········existingPublicKey,⏎········existingKid·||·crypto.randomUUID()⏎······`

(prettier/prettier)